# this message contains information for setting encoder positions
int32[] motor_number
int32[] accel
int32[] speed
int32[] deccel
float32[] angle
